### PR TITLE
Add the ability to disable linter if no .eslintrc file is found.

### LINKED ISSUE #74: Ability to disable linting if no .eslintrc file found.
linter-jslint has the ability to disable linting if it doesn't fine the .jslintrc file. Would be cool if This package did the same.

### PR DESCRIPTION
**Pull request by digitalfiz**
This PR fixes #74

### CODE REVIEW

File: lib/index.js [hunk 1]
```diff
@@ -33,6 +49,11 @@ export default {
         const filePath = TextEditor.getPath();
         const fileText = TextEditor.getText();
         const ignoreFile = helpers.find(filePath, '.eslintignore');
+        const eslintrc = helpers.find(filePath, ['.eslintrc', '.eslintrc.js', '.eslintrc.json', '.eslintrc.yml']);
+
+        if (this.disableWhenNoEslintrcFileInPath && !eslintrc) {
```
Comment (Adezandee) [inline lib/index.js]: I would simply use `atom.config.get('linter-eslint. disableWhenNoEslintrcFileInPath')` instead of registering an event handler

**Comment by Adezandee** [comment]
Hi @digitalfiz, thanks for your PR. can you please format your commit message following angular semantic release format and make the CI happy ? 👍

**Comment by digitalfiz** [comment]
Yup,  will fix in a few and I will also update with your suggestion of atom.config.get

**Comment by Adezandee** [comment]
Moreover, I'm kind of annoyed by this addition because it adds a configuration parameter even so this project claims `zeroconf` from the beginning. I understand your problem with the linter kicking in even with a not wet configured project or single files, but the goal here is to be as close as possible from command-line `eslint`. I know that deactivating a plugin every time is a chore, but the `eslint` way to this problem maybe is `$ echo "*" >> .eslintignore` at the project root ?

**Comment by digitalfiz** [comment]
I understand no worries. I found linter-eslint which more aligns with the other linter-* packages I have installed and it has the configuration option. I'll delete this PR as it conflicts with the goals of the project.

**Pull request closed without merging**